docs/reference/cluster/nodes-stats.asciidoc

```diff
@@ -834,6 +834,156 @@ This is not shown for the `shards` level, since mappings may be shared across th
 
 =======
 
+`shards`::
+(object) Contains statistics aggregated at the `shards` level, per index, as well as shard-specific statistics.
++
+.Properties of `shards`
+[%collapsible%open]
+=======
+
+`<index_name>`::
+(array of objects) Contains a list of all shards for the index.
++
+.Properties of `<index_name>`
+[%collapsible%open]
+========
+
+`<shard_id>`::
+(object) Contains statistics for the shard.
++
+.Properties of `<shard_id>`
+[%collapsible%open]
+=========
+
+`routing`::
+(object) Contains routing information about the shard instance.
++
+.Properties of `routing`
+[%collapsible%open]
+==========
+
+`state`::
+(string) State of the shard. Returned values are:
++
+* `INITIALIZING`: The shard is recovering from a peer shard or gateway.
```

Comment:
```suggestion
* `INITIALIZING`: The shard is initializing/recovering.
```

Reply:
I actually borrowed the phrasing for these stats from [here](https://github.com/elastic/elasticsearch/blob/fd916c221ae51b621dc5eb4d93b8f5a10bbc6d49/docs/reference/cat/shards.asciidoc?plain=1#L72), should I still change it here? (Or change it in both spots?)

Reply:
I'd prefer what I suggested, or simply `recovering`, just to avoid readers wondering over `gateway`.